Haskell/MaîtreD.hs

```diff
@@ -1,11 +1,19 @@
 module MaîtreD where
 
-import Data.Time (ZonedTime)
+import Data.Time (ZonedTime (zonedTimeToLocalTime, zonedTimeZone))
```

Comment:
Yes, actually, [searching for `ZonedTime` on GitHub](https://github.com/search?utf8=%E2%9C%93&q=ZonedTime&type=Code&ref=searchresults) yields quite a few results with `ZonedTime (..)`.

Since we're using `ZonedTime (..)` in MaîtreDTests module, I'll go ahead and change also this one to  `ZonedTime (..)`.

Problem solved, I think. :relieved: